packages/wallets/src/features/cfd/screens/Jurisdiction/JurisdictionTncSection/JurisdictionTncSection.scss

```
@@ -0,0 +1,33 @@
+.wallets-jurisdiction-tnc {
+    display: flex;
+    flex-direction: column;
+    gap: 1.2rem;
+    justify-content: center;
+    margin-top: 3rem;
+
+    @include mobile {
+        position: sticky;
+        bottom: 0;
+        background: #ffffff;
```

it will be easier if we decided to make colour tweaks in the future
```suggestion
        background: var(--system-light-8-primary-background, #fff);
```